src/components/NoteEditor.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <textarea
+    name="editor"
+    rows="18"
+    autocomplete="off"
+    class="textarea is-primary"
+    placeholder="何を書きますか？"
+    v-model="localNote.body"
+    ref="editor"
+  />
+  <div class="columns mt-3">
+    <div class="column">
+      <button
+        v-if="note.id > -1"
```

Comment:
```suggestion
        v-if="isPersisted"
```

みたいなcomputedを導入した方がわかりやすいかも。

あと、`note.id > -1`は`note.id !== undefined`みたいに書いた方が理解しやすい気もします。
`> -1`と書くと「-1もあり得るが除外する」という意味に見えるので。

Reply:
たしかに`v-if`の中で処理をしてしまってるので、computedにするべきでした。 1111bd5
また、`undefined`を適切に活用するように修正しました 🙇‍♂️